go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/DelineaXPM/tss-sdk-go/v2
 
 go 1.13
+
```

Comment:
@pacificcode are we using these dependencies? it is showing me unused references.